Dockerfile.ubuntu

```diff
@@ -103,8 +103,8 @@ RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
       cuda-11-8 \
```

Comment:
Could we add explicit, currently used versions for other cuda packages (prevent problems in future)